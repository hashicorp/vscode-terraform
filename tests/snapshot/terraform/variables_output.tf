output "instance_ip_addr" {
  value = aws_instance.server.private_ip
}

output "instance_ip_addr" {
  value       = aws_instance.server.private_ip
  description = "The private IP address of the main server instance."

  depends_on = [
    aws_security_group_rule.local_access,
  ]
}
