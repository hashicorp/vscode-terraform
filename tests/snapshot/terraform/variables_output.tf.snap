>output "instance_ip_addr" {
#^^^^^^ source.terraform meta.block.terraform entity.name.type.terraform
#      ^ source.terraform meta.block.terraform
#       ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.begin.terraform
#        ^^^^^^^^^^^^^^^^ source.terraform meta.block.terraform entity.name.tag.terraform
#                        ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.end.terraform
#                         ^ source.terraform meta.block.terraform
#                          ^ source.terraform meta.block.terraform punctuation.section.block.begin.terraform
>  value = aws_instance.server.private_ip
#^^ source.terraform meta.block.terraform
#  ^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#       ^ source.terraform meta.block.terraform variable.declaration.terraform
#        ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#         ^ source.terraform meta.block.terraform variable.declaration.terraform
#          ^^^^^^^^^^^^ source.terraform meta.block.terraform
#                      ^ source.terraform meta.block.terraform keyword.operator.accessor.terraform
#                       ^^^^^^ source.terraform meta.block.terraform variable.other.member.terraform
#                             ^ source.terraform meta.block.terraform keyword.operator.accessor.terraform
#                              ^^^^^^^^^^ source.terraform meta.block.terraform variable.other.member.terraform
>}
#^ source.terraform meta.block.terraform punctuation.section.block.end.terraform
>
>output "instance_ip_addr" {
#^^^^^^ source.terraform meta.block.terraform entity.name.type.terraform
#      ^ source.terraform meta.block.terraform
#       ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.begin.terraform
#        ^^^^^^^^^^^^^^^^ source.terraform meta.block.terraform entity.name.tag.terraform
#                        ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.end.terraform
#                         ^ source.terraform meta.block.terraform
#                          ^ source.terraform meta.block.terraform punctuation.section.block.begin.terraform
>  value       = aws_instance.server.private_ip
#^^ source.terraform meta.block.terraform
#  ^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#       ^^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform
#              ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#               ^ source.terraform meta.block.terraform variable.declaration.terraform
#                ^^^^^^^^^^^^ source.terraform meta.block.terraform
#                            ^ source.terraform meta.block.terraform keyword.operator.accessor.terraform
#                             ^^^^^^ source.terraform meta.block.terraform variable.other.member.terraform
#                                   ^ source.terraform meta.block.terraform keyword.operator.accessor.terraform
#                                    ^^^^^^^^^^ source.terraform meta.block.terraform variable.other.member.terraform
>  description = "The private IP address of the main server instance."
#^^ source.terraform meta.block.terraform
#  ^^^^^^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#             ^ source.terraform meta.block.terraform variable.declaration.terraform
#              ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#               ^ source.terraform meta.block.terraform variable.declaration.terraform
#                ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.terraform meta.block.terraform string.quoted.double.terraform
#                                                                    ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>
>  depends_on = [
#^^ source.terraform meta.block.terraform
#  ^^^^^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#            ^ source.terraform meta.block.terraform variable.declaration.terraform
#             ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#              ^ source.terraform meta.block.terraform variable.declaration.terraform
#               ^ source.terraform meta.block.terraform punctuation.section.brackets.begin.terraform
>    aws_security_group_rule.local_access,
#^^^^ source.terraform meta.block.terraform
#    ^^^^^^^^^^^^^^^^^^^^^^^ source.terraform meta.block.terraform variable.other.readwrite.terraform
#                           ^ source.terraform meta.block.terraform keyword.operator.accessor.terraform
#                            ^^^^^^^^^^^^ source.terraform meta.block.terraform variable.other.member.terraform
#                                        ^ source.terraform meta.block.terraform punctuation.separator.terraform
>  ]
#^^ source.terraform meta.block.terraform
#  ^ source.terraform meta.block.terraform punctuation.section.brackets.end.terraform
>}
#^ source.terraform meta.block.terraform punctuation.section.block.end.terraform
>